source/office365/index.rst

```diff
@@ -10,16 +10,16 @@ Using Wazuh to monitor Office 365
 
 .. versionadded:: 4.3.0
 
-This section provides instructions for monitoring the **Office 365** audit log for your organization.
+This section provides instructions for monitoring the Office 365 ``audit log`` for your organization.
 
-The **audit** log allows Wazuh to monitor:
+The ``audit log`` allows Wazuh to monitor:
 
 - User activity in SharePoint Online and OneDrive for Business.
 - User activity in Exchange Online (Exchange mailbox audit logging).
 - Admin activity in SharePoint Online.
 - Admin activity in Azure Active Directory (the directory service for Microsoft 365).
 - Admin activity in Exchange Online (Exchange admin audit logging).
-- eDiscovery activities in the security and compliance center.
+- Discovery activities in the security and compliance center.
```

Comment:
```suggestion
- eDiscovery activities in the security and compliance center.
```

Reply:
eDiscovery is correct: https://learn.microsoft.com/en-us/microsoft-365/compliance/ediscovery?view=o365-worldwide